packages/z3/z3.4.8.13/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "weng@cs.jhu.edu"
+authors: "MSR"
+homepage: "https://github.com/Z3prover/z3"
+bug-reports: "https://github.com/Z3prover/z3/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/Z3prover/z3.git"
+depexts: [
+  ["python3-distutils"] {os-family = "debian"}
+]
+build: [
+  [ "python3" "scripts/mk_make.py" "--ml" ]
+  [ make "-C" "build" "-j" jobs ]
+]
+
+install: [
+  [ "sh" "-c" "ocamlfind install z3 build/api/ml/* -dll build/libz3.*"]
+  [ "cp" "build/z3" "%{bin}%/z3"]
+]
+
+depends: [
+  "ocaml"
+  "ocamlfind" {build}
+  "zarith"
+  "conf-gmp"
+  "conf-python-3" {build}
+]
+x-ci-accept-failures: [
+  "centos-7" "oraclelinux-7" # C compiler is too old
+]
+synopsis: "Z3 solver"
```

Comment:
If this is to be merged in the current state, I believe it at least needs to be a message to tell the users about this silent breaking change. Something like this:
```suggestion
synopsis: "Z3 solver"
post-messages: [
  """
This version of Z3 changed the linking mode from static to dynamic.
This change is silent and potentially breaking.
Please make sure that you don’t require static binaries in the programs using Z3.
""" {success}
]
```